deepchem/feat/basic.py

```diff
@@ -69,9 +77,12 @@ class RDKitDescriptors(Featurizer):
   ])
 
   def __init__(self):
+    try:
+      from rdkit.Chem import Descriptors
+    except ModuleNotFoundError:
+      raise ValueError("This class requires RDKit to be installed.")
     self.descriptors = []
     self.descList = []
-    from rdkit.Chem import Descriptors
     for descriptor, function in Descriptors.descList:
       if descriptor in self.allowedDescriptors:
         self.descriptors.append(descriptor)
```

Comment:
self.descriptors is not used?

Reply:
Good catch as well! I think this old code that hasn't been scrutinized in a while. Will fix